server/scsynth/SC_UnitDef.cpp

```diff
@@ -22,6 +22,7 @@
 
 #include "SC_Endian.h" // first to avoid win32 IN clash
 
+#include "SC_Errors.h"
```

Comment:
What was this header needed for?

Reply:
good catch, thanks a lot! it was just some LSP byproduct of previous work. I've removed it now